src/main/java/frc/robot/Constants.java

```diff
@@ -334,6 +335,32 @@ public static class PoseConstants {
         public static final Pose2d REEFSCORE5_2 = new Pose2d(3.101, 4.175, new Rotation2d(0));
         public static final Pose2d REEFSCORE6_1 = new Pose2d(3.656, 5.122, new Rotation2d(300));
         public static final Pose2d REEFSCORE6_2 = new Pose2d(3.949, 5.282, new Rotation2d(300));
+
+        public enum poses {
```

Comment:
capital and rename this something a little less vague (yeah ik its self explanatory but still)